docs/contributing.rst

```diff
@@ -34,10 +34,13 @@ If you have questions about the disposition of a request, feel free to email one
 Core Committer Team
 ~~~~~~~~~~~~~~~~~~~~~
 
-* Ben.BroderickPhillips@emc.com
-* Felix.Yuan@emc.com
-* Andrew.Hou@emc.com
-* Brian.Parry@emc.com
+* Jeanne.Ohren@dell.com
+* Felix.Yuan@dell.com
+* Andrew.Hou@dell.com
+* Brian.Parry@dell.com
+* Rahman.Muhammad@dell.com
```

Comment:
I have Rahman_Muhammad@dell.com in my emails.  Unless there are multiple aliases.